### PR TITLE
New blog: Chaos Engineering - Breaking things Intentionally

### PR DESCRIPTION
**Pull request by manishdangi98**
Signed-off-by: manishdangi98 <manish1998dangi@gmail.com>

### CODE REVIEW

**Comment by milasuperstar** [comment]
@dcalvin

File: blog/2020-10-30-chaos-engineering-breaking-things-intentionally.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+- To improve  resiliency of the system
+- To  expose hidden threats and vulnerability of the system
+- To figure out system weaknesses before they cause any failure in production 
+
+Lots of people think that they are not as big compared to Netflix and other tech giants; nor do they have any  databases or systems of that scale.
+
+They are probably right, but over the period, Chaos engineering has evolved so much that it’s no longer limited to digital companies like Netflix. To ensure consistent performance and constant availability of their systems, more and more companies from different industries are implementing chaos experiments. 
+
+##Chaos-Mesh
```
Comment (dcalvin) [inline blog/2020-10-30-chaos-engineering-breaking-things-intentionally.md:36]:
```suggestion
## Chaos-Mesh
```

File: blog/2020-10-30-chaos-engineering-breaking-things-intentionally.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- **pod-failure:** Simulates continuous unavailability of Kubernetes Pods
+- **pod-kill:** Simulates the Kubernetes Pod being killed
+
+Chaos Mesh mainly focuses on the simplicity of how all chaos tests are done quickly and easily understandable to anyone using it.
+
+The recent [1.0 release](https://chaos-mesh.org/blog/chaos-mesh-1.0-chaos-engineering-on-kubernetes-made-easier/) provides the general availability of Chaos Dashboard, which  Chaos simplifies the complexities of chaos experiment. With a few mouse clicks, you can define the Chaos experiment's scope, specify the type of chaos injection, define scheduling rules, and observe the chaos experiment results- all in the dashboard of Chaos Mesh.
+
+In case you want to try Chaos Mesh in your browser, checkout [Katakoda interactive tutorial](https://chaos-mesh.org/interactiveTutorial/), where you can get your hands on  Chaos Mesh without even deploying it. To understand the design principles and how Chaos Mesh works, read [this blog](https://chaos-mesh.org/blog/chaos_mesh_your_chaos_engineering_solution) by the project's maintainer, [Cwen Yin](https://www.linkedin.com/in/cwen-yin-81985318b/).
+
+##Join the community
```
Comment (dcalvin) [inline blog/2020-10-30-chaos-engineering-breaking-things-intentionally.md:64]:
```suggestion
## Join the community 
```

**Review by g1eny0ung — CHANGES_REQUESTED** [review]

**Review by dcalvin — APPROVED** [review]
LGTM. Nice work @manishdangi98

**Review by g1eny0ung — APPROVED** [review]

**Pull request merged**